php/src/Container/IContainerState.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace AIO\Container;
+
+enum IContainerState: string
```

Comment:
```suggestion
enum ContainerState: string
```

The `I` prefix is usually reserved for interfaces and it can be dropped in my opinion.